packages/curve-ui-kit/src/themes/design/0_primitives.ts

```diff
@@ -104,6 +104,7 @@ export const Duration = {
   Tooltip: { Enter: 500, Exit: 500 },
   Flicker: 1000,
   FormDebounce: 500,
+  sliderDebounce: 300,
```

Comment:
the other vars in the design system use a starting capital :nerd_face: 
```suggestion
  SliderDebounce: 300,
```